TYPO3.Flow/Classes/TYPO3/Flow/Core/Bootstrap.php

```diff
@@ -590,6 +602,12 @@ protected function ensureRequiredEnvironment()
                 exit(1);
             }
         }
+        if (!is_dir(FLOW_PATH_TEMPORARY) && !is_link(FLOW_PATH_TEMPORARY)) {
+            if (!@mkdir(FLOW_PATH_TEMPORARY, 0777, true)) {
```

Comment:
I see, seems okay then. However this might not work as expected since it doesn't do the umask workaround like ``createDirectoryRecursively `` does, which could lead to a different file permission being set. Why not use a try catch statement?